README.md

```diff
@@ -1,2 +1,95 @@
 # CapstoneProject2
-Javascript Capstone Project 2
+
+RECIPE HUB - Javascript capstone project 
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# RECIPE HUB PROJECT
+
+This is the capstone project for module two in Microverse (Kanban board).
+In this project we created a fully functional web app that fetches tv shows from an api [Meal App](https://www.themealdb.com)
+Users can also comment and like movies and shows.
+
+### [Project Demo Link](https://www.loom.com/share/e59aee87611948e5961a0c344acc8d61)
```

Comment:
- Please add another link to the live demo link of the deployed version of the app besides the project presentation. Both links should be listed in the README file 🙏